server/web/__init__.py

```diff
@@ -99,15 +99,15 @@ def bad_request_handler(e):
     logistics_wizard.errorhandler(404)(not_found_handler)
 
     # Register app with Service Discovery and initiate heartbeat cycle if running in PROD
-    if Config.ENVIRONMENT == 'PROD' and Config.SD_STATUS == 'ON' and env.get('VCAP_APPLICATION') is not None:
+    if Config.SD_STATUS == 'ON' and env.get('VCAP_APPLICATION') is not None:
         from signal import signal, SIGINT, SIGTERM
         from sys import exit
 
         # Create service publisher and register service
         creds = json.loads(env['VCAP_SERVICES'])['service_discovery'][0]['credentials']
-        publisher = ServicePublisher('lw-controller', 300, 'UP',
+        publisher = ServicePublisher('lw-controller', 30, 'UP',
                                      '%s.mybluemix.net' % json.loads(env['VCAP_APPLICATION'])['name'],
```

Comment:
Agreed for app name in our context. But in EU, name would stay logistics-wizard however domain will be eu-gb.mybluemix.net and there we would have a bug no? Using the URI would fix this one.